systemtests/src/main/java/io/enmasse/systemtest/messagingclients/MessagingClientRunner.java

```diff
@@ -105,6 +119,69 @@ public void sendAndReceive(MessagingEndpoint endpoint, boolean waitReceivers, Ma
         }
     }
 
+    public void endpointSendAndReceiveOnCluster(String host, int port, String address, boolean enableTls, boolean websockets) throws InterruptedException {
```

Comment:
please change name, I don't understand why it is named endpointSendEndReceive

Reply:
Changed, but I am thinking that we could maybe squash sendAndReceive with this method in some way. Need to take a look on that little bit more.